device_drivers/JsonToDriver.py

```diff
@@ -1,7 +1,25 @@
-from FileOutput import WriteDriver
-from AvalonJsonParser import AvalonJsonParse
+from .driver_config import DriverConfig
+from .device import get_device
```

Comment:
I'm not familiar with the .<package> import statement.  Is this some form of "relative" import?

Reply:
Yes, that is exactly what is. It does require that your `__main__` file isn't in the same directory because it is relative from the package, not the file system and the package isn't defined when when the `__main__` file is in the same directory. So because `autogen_device_driver.py` is in the root directory and not `device_drivers` it can do relative imports